### PR TITLE
Improve logger

### PR DESCRIPTION
**Pull request by well1791**

### CODE REVIEW

File: notification-quickstart.iml [hunk 1]
```diff
@@ -10,7 +10,7 @@
     </content>
```
Comment (hortega) [inline notification-quickstart.iml]: I think we could add this file to `.gitignore`

**Review by hortega — CHANGES_REQUESTED** [review]
Looking good @well1791 . There is a call to Twilio library in `NotifyUtil` likely to throw an exception, could you catch it and log it as per the story?

File: notification-quickstart.iml (hunk 1 above)
Comment (well1791) [reply notification-quickstart.iml]: thanks for the reviewing! I'll update this change on all repos I've worked on.

**Comment by joliveros** [comment]
LGTM @well1791

**Pull request merged**